test/framework/clusterctl/repository.go

```diff
@@ -149,8 +149,13 @@ func YAMLForComponentSource(ctx context.Context, source ProviderVersionSource) (
 		}
 		data = buf
 	case KustomizeSource:
+		// Set Path of kustomize binary using KUSTOMIZE_PATH env
+		kustomizePath, ok := os.LookupEnv("KUSTOMIZE_PATH")
```

Comment:
What do we think about namespacing the env var?
We currently have:
* CAPI_KIND_CLUSTER_NAME
* CAPI_WEBHOOK_HOSTNAME
* CAPI_DISABLE_TEST_ENV

So this could be `CAPI_KUSTOMIZE_PATH`. I'm personally in favor of namespacing env vars like this so CAPI doesn't pick up env vars set for an entirely different reason by accident

Reply:
okay will do

Reply:
Might be good to wait for other opinions to avoid back and forth :)